components/brave_wallet_ui/page/screens/onboarding/onboarding.routes.tsx

```diff
@@ -104,6 +105,14 @@ export const OnboardingRoutes = () => {
         </Route>
       }
 
+      <Route path={WalletRoutes.OnboardingConnectHardwareWalletStart || WalletRoutes.OnboardingConnectHardwareWallet} exact>
```

Comment:
This will always look for `WalletRoutes.OnboardingConnectHardwareWalletStart` since it is a truthy value